src/lazy/encoding.rs

```diff
@@ -241,51 +242,51 @@ impl Encoding for TextEncoding_1_1 {
 }
 
 /// Marker trait for binary encodings of any version.
-pub trait BinaryEncoding<'top>: Encoding<Output = Vec<u8>> + Decoder {}
+pub trait BinaryEncoding: Encoding<Output = Vec<u8>> + Decoder {}
 
 /// Marker trait for text encodings.
-pub trait TextEncoding<'top>:
+pub trait TextEncoding:
```

Comment:
🪧 The rest of the changes in this file move the lifetime to the method signatures.